### PR TITLE
feat: support eggScriptsConfig in package.json

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,20 @@ $ eggctl stop [--title=example]
 - **Options**
   - `title` - process title description, use for kill grep.
 
+## Options in `package.json`
+
+In addition to the command line specification, options can also be specified in `package.json`. However, the command line designation takes precedence.
+
+```json
+{
+  "eggScriptsConfig": {
```
Comment (fengmk2) [inline README.md]: 是否改成 config 节点下比较好？

```json
{
  "config": {
    "eggScripts": { ... }
  }
}
```

Comment (dead-horse) [reply README.md]: 现在有 egg, eggPlugin 等一系列配置了，还是先平铺吧？

**Comment by dead-horse** [comment]
2.12.0

File: lib/command.js [hunk 2]
```diff
@@ -47,6 +47,13 @@ class Command extends BaseCommand {
       if (pkgInfo && pkgInfo.egg && pkgInfo.egg.typescript) {
         argv.sourcemap = true;
       }
+
+      // read argv from eggScriptsConfig in package.json
+      if (pkgInfo && pkgInfo.eggScriptsConfig && typeof pkgInfo.eggScriptsConfig === 'object') {
+        for (const key in pkgInfo.eggScriptsConfig) {
+          if (argv[key] == null) argv[key] = pkgInfo.eggScriptsConfig[key];
```
Comment (atian25) [inline lib/command.js:54]: 结合 https://github.com/node-modules/common-bin/blob/master/lib/helper.js#L180

应该可以通过 node-options--max-http-header-size 的方式来注入 Node 启动参数

**Pull request merged**